engine/src/main/scala/cromwell/server/WorkflowManagerSystem.scala

```diff
@@ -23,5 +23,5 @@ trait WorkflowManagerSystem {
   CromwellBackend.initBackends(allowedBackends, defaultBackend, actorSystem)
   // For now there's only one WorkflowManagerActor so no need to dynamically name it
   lazy val workflowManagerActor = actorSystem.actorOf(WorkflowManagerActor.props(), "WorkflowManagerActor")
-  lazy val validateActor = actorSystem.actorOf(Props[ValidateActor], "ValidationActor")
+  lazy val workflowDescriptorMaterializeActor = actorSystem.actorOf(Props[MaterializeWorkflowDescriptorActor], "MaterializeWorkflowDescriptorActor")
```

Comment:
Let's please not use this reflectiony form of `actorOf` if we can avoid it, as we saw last week this can break things in ways unit tests don't catch.